opentreemap/importer/errors.py

```diff
@@ -3,13 +3,18 @@
 from __future__ import unicode_literals
 from __future__ import division
 
-# TODO: stop storing error text in the database in order to
-# make lazy translation possible.
-# See https://github.com/OpenTreeMap/OTM2/issues/1880
-# from django.utils.translation import ugettext_lazy as trans
-trans = lambda x: x
+from django.utils.translation import ugettext_lazy as trans
+
+_messages_by_code = {}
+
+def e(code, message, fatal):
+    _messages_by_code[code] = message
+    return (code, message, fatal)
+
+
+def message(code):
```

Comment:
consider renaming this to `get_message`, because `error['msg']` and `errors.message` look deceptively similar despite their semantic difference.